### PR TITLE
docs(en-US): list and search users

### PR DESCRIPTION
**Pull request by IceHe**
Manage users on the AC page

- List users
- Search users

image

### CODE REVIEW

**Comment by IceHe** [comment]
[LOG-3169 Summary "Manage users on the Admin Console page"](https://linear.app/silverhand/issue/LOG-3169/manage-users-on-the-admin-console-page-summary)

File: docs/docs/recipes/manage-users/on-admin-console-page.md [hunk 1]
```diff
@@ -5,6 +5,25 @@ sidebar_position: 1
 
 # Manage users on the Admin Console page
 
+In Admin Console, click the **User Management** tab on the left, and you'll see a user table view on the right.
```
Comment (gao-sun) [inline docs/docs/recipes/manage-users/on-admin-console-page.md:8]: you need a specific mission or scenario for a recipe, otherwise it'll be a reference

Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md:10]: updated. PTAL

**Comment by IceHe** [comment]
[LOG-3169 List users](https://linear.app/silverhand/issue/LOG-3169/list-users)

File: docs/docs/recipes/manage-users/on-admin-console-page.md [hunk 2]
```diff
@@ -5,6 +5,29 @@ sidebar_position: 1
...
+
+In Admin Console, click the **User Management** tab on the left, and you'll see a user table view on the right.
+
+There are three columns in the table:
+
+- **User**: The user info includes avatar, name (for display), and username (for sign-in).
+- **From application**: The name of the application from which the user is.
+- **Latest sign in**: The timestamp when the user signed in last time.
+
+To navigate users, click the "<" next page button, the ">" previous page button, or the specific page number at the bottom right corner.
```
[the remark this answers is not in the document]
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md]: Fixed

> To navigate users, click the "<" previous page button, the ">" next page button, or the specific page number at the bottom right corner.

File: docs/docs/recipes/manage-users/on-admin-console-page.md (hunk 1 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md]: Updated. PTAL

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import GotoUserManagement from './fragments/_goto-user-management.md';
+
+## List users
+
+<GotoUserManagement />
+
+There are three columns in the table:
+
+- **User**: The user info includes avatar, name (for display), and username (for sign-in).
+- **From application**: The name of the application from which the user is.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.mdx:17]: the user is initially registered?

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+- **Latest sign in**: The timestamp when the user signed in last time.
+
+To navigate users, click the "<" previous page button, the ">" next page button, or the specific page number at the bottom right corner.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.mdx]: `navigate users` sounds not right

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+import GotoUserManagement from './fragments/_goto-user-management.md';
+
+## List users
+
+<GotoUserManagement />
+
+There are three columns in the table:
+
+- **User**: The user info includes avatar, name (for display), and username (for sign-in).
```
Comment (darcyYe) [inline docs/docs/recipes/manage-users/on-admin-console-page.mdx:16]: I can hardly distinguish username and name from the UI until I see `Unnamed`. Maybe give more detailed instructions.

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 3 above)
Comment (darcyYe) [inline docs/docs/recipes/manage-users/on-admin-console-page.mdx:17]: Can't understand.

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- **From application**: The name of the application from which the user is.
+- **Latest sign in**: The timestamp when the user signed in last time.
+
+To navigate users, click the "<" previous page button, the ">" next page button, or the specific page number at the bottom right corner.
+
+## Search users
+
+<GotoUserManagement />
```
Comment (darcyYe) [inline docs/docs/recipes/manage-users/on-admin-console-page.mdx:22]: Not sure the reading experience is smooth if we always start with the same instructions. Maybe reword it.

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+Input a keyword in the search box above the user table, click the "Search" button, and you'll see the matching users in the table.
+
+The keyword can be:
```
Comment (darcyYe) [inline docs/docs/recipes/manage-users/on-admin-console-page.mdx:26]: Maybe need to show that these keyword matches should be an exact match, not a partial match?

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+- name,
+- username,
+- email,
+- or phone number.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.mdx:31]: Fuzzy search is supported on the following user properties:

- name
- username
- email
- phone number

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 5 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:16]: Maybe too detailed?

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 7 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:26]: just my personal opinion, maybe its implementation will change soon, seems that no need to explode too many details in implementation.

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 4 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx]: Removed.

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 8 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:31]: exact and partial match for now.

File: docs/docs/recipes/manage-users/fragments/_goto-user-management.md [hunk 9]
```diff
@@ -0,0 +1 @@
+In Admin Console, click the **User Management** tab on the left, and you'll see a user table view on the right.
```
[the remark this answers is not in the document]
Comment (IceHe) [reply docs/docs/recipes/manage-users/fragments/_goto-user-management.md:1]: The description aligns with the current doc `docs/docs/recipes/customize-sie/README.md` for now.

image

Maybe need further discussion to agree on how to describe the element on the AC page.

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+
+# Manage users on the Admin Console page
+
+import GotoUserManagement from './fragments/_goto-user-management.md';
+
+## List users
+
+<GotoUserManagement />
+
+There are three columns in the table:
```
[the remark this answers is not in the document]
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:14]: seems that `in the table` is enough

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:16]: Updated

```markdown
- **User**: The user info includes the avatar, full name, and username.
```

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 3 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:17]: Updated. PTAL

```md
- **From application**: The name of the application that the user initially registered.
```

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 8 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:31]: Updated. PTAL

```md
Support the exact partial search on the following user properties:
```

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 7 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:26]: Updated. PTAL

```md
Support the exact partial search on the following user properties:
```

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 3 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:17]: Updated. PTAL

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 6 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx:22]: Maybe need further discussion. Maybe reorg later?

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 5 above)
Comment (darcyYe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx]: We discussed this on [slack](https://silverhand-io.slack.com/archives/C0243P3E4KS/p1656039242323229) this morning, and the user column might show email, phone or username, and users' full name. Please align this before merging.

**Review by darcyYe — APPROVED** [review]
Overall LGTM.

File: docs/docs/recipes/manage-users/on-admin-console-page.mdx (hunk 5 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.mdx]: Updated. PTAL

File: docs/docs/recipes/manage-users/on-admin-console-page.md (hunk 1 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md]: Updated. PTAL.

File: docs/docs/recipes/manage-users/using-management-api.md [hunk 11]
```diff
@@ -3,6 +3,6 @@ sidebar_label: Manage by Core management API
 sidebar_position: 2
 ---
 
-# Manage users by Logto Core management API
+# Manage users Using Management API
```
Comment (gao-sun) [inline docs/docs/recipes/manage-users/using-management-api.md:6]:
```suggestion
# Manage users using Management API
```

File: docs/docs/recipes/manage-users/using-admin-console.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+- **Latest sign in**: The timestamp when the user signed in last time.
+
+Input a keyword in the search box above the user table, click the "Search" button, and you'll see the matching users in the table.
+
+Support the exact partial search on the following user properties:
+
+- name,
+- username,
+- email,
+- or phone number.
```
[the remark this answers is not in the document]
Comment (IceHe) [reply docs/docs/recipes/manage-users/using-admin-console.md:25]: 1. It should be `partial search`
2. I'm good with both. Intended to stand out which field can be a keyword. Rewrote in one line.
